package.json

```diff
@@ -38,6 +38,7 @@
     "moment": "^2.15.2",
     "morgan": "~1.7.0",
     "nodemailer": "~2.6.4",
+    "pg": "^5.1.0",
```

Comment:
We already had `pg` as a transitive dependency from `pg-promise`, but I actually interacted with it to change return types from postgres [here](https://github.com/beloitcollegecomputerscience/OED/pull/79/files#diff-089faff11224ef4b4082159c0a2b9d40). I thought it was a good idea to add it as an explicit dependency since we're explicitly using it.